docs/node-groups.md

```diff
@@ -744,3 +744,7 @@ For example, if you are enabling the `metrics-server` Kubernetes add-on (Helm ch
     ]
   }
 ```
+
+### Security groups attached to self-managed node groups
+
+EKS automatically creates a [cluster security group](https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.html) and attaches it to any managed node groups in the cluster. This framework attaches the EKS cluster security group along with an additional security group to self-managed node groups, so that they can communicate with any EKS managed node groups that may be present in the cluster. As recommended in the AWS Load Balancer Controller [troubleshooting guide](https://aws.amazon.com/premiumsupport/knowledge-center/load-balancer-troubleshoot-creating/), the framework ensures that only one of the security groups - the EKS cluster security group - is tagged with tag `kubernetes.io/cluster/[cluster-name] = "owned"`, so that AWS Load Balancer Controller can create necessary rules for ingresses in the EKS cluster security group.
```

Comment:
the cluster primary security group is only attached to nodes in the following scenarios:
1. Fargate profiles 
2. The default launch template is used for EKS managed node groups

Reply:
Looks like there is some confusion here. This README change is a description of *this particular change in the PR*. After the PR is merged, the README text will be valid - i.e. this framework (the Terraform AWS EKS Blueprints module) will attach EKS cluster security group to self-managed node groups.